### PR TITLE
MAINT: set FixedBasisPointsSlippage as default

### PR DESCRIPTION
**Pull request by vikram-narayan**
set `FixedBasisPointsSlippage` as default slippage model

### CODE REVIEW

File: zipline/algorithm.py [hunk 1]
```diff
@@ -320,6 +321,7 @@ def __init__(self, *args, **kwargs):
         if not self.blotter:
             self.blotter = Blotter(
                 data_frequency=self.data_frequency,
+                equity_slippage=FixedBasisPointsSlippage(),
```
Comment (ssanderson) [inline zipline/algorithm.py]: How were we setting the default slippage model before if we weren't passing it here?  Should we be doing this there instead?

**Comment by vikram-narayan** [comment]
changed the default setting to be in `blotter.py`, and added params to make equity and future slippage models configurable

File: zipline/algorithm.py (hunk 1 above)
Comment (vikram-narayan) [reply zipline/algorithm.py]: it was set in blotter before, changed the default to be set in blotter, but made it configurable from `TradingAlgorithm`

**Comment by ssanderson** [comment]
@vikram-narayan changes here look good to me, but I think you've still got some real test failures from updating the slippage model.

**Comment by vikram-narayan** [comment]
@ssanderson made the slippage model return `None, None` if the number of shares to fill is 0. Appveyor is passing, still seeing the weird issue with travis builds

File: zipline/algorithm.py [hunk 2]
```diff
@@ -250,6 +250,12 @@ def __init__(self, *args, **kwargs):
             identifiers : List
                 Any asset identifiers that are not provided in the
                 equities_metadata, but will be traded by this TradingAlgorithm
+            equity_slippage : SlippageModel
```
Comment (ssanderson) [inline zipline/algorithm.py]: Is there a reason to prefer making this an arg vs. using the existing `set_slippage` API function?  It seems a little confusing to have two different ways to do this.

File: zipline/examples/__init__.py [hunk 3]
```diff
@@ -77,6 +78,8 @@ def run_example(example_name, environ):
         analyze=getattr(mod, 'analyze', None),
         bundle='test',
         environ=environ,
+        # example test data use the VolumeShareSlippage model
+        equity_slippage=VolumeShareSlippage(),
```
Comment (ssanderson) [inline zipline/examples/__init__.py]: Setting this here makes the examples a little bit confusing; if you ran the example code exactly as defined in the example file, you'd get a different result than what we produce here.  It looks like we call `set_commission` in most of the examples for this reason.

**Comment by ssanderson** [comment]
LGTM

**Pull request merged**